src/main/resources/application-han.yml

```
@@ -39,3 +39,7 @@ spring:
     redis:
       host: localhost
       port: 6379
+
+discord:
+  alert:
+    enabled: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)